rubocop-airbnb/config/rubocop-style.yml

```diff
@@ -347,8 +347,9 @@ Style/FrozenStringLiteralComment:
   Enabled: false
```

Comment:
Can we enable this now we're not supporting 2.3? I remember it caused some problems when I updated the version a while ago but I wonder if it would be alright now

Reply:
@zpaines great question. Given the delay in requiring frozen string literals beyond the ruby 3 release (see [some rubocop discussion on this topic](https://github.com/rubocop/rubocop/issues/7197)), I'm less convinced that this is pressing now. There are potentially good reasons to want to enable frozen string literals, say for performance reasons, but I'm not sure where I land on enforcing this in the style guide now. Let's talk about this when we evaluate any new cops that have been added to rubocop since our last update and make a decision about this going forward for a 4.1.0 release